pkg/controller/atlasdatafederation/private_endpoint.go

```diff
@@ -3,24 +3,28 @@ package atlasdatafederation
 import (
 	"context"
 
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/translation/datafederation"
```

Comment:
< nit > Move down to other AKO dependancies

Reply:
fixed 👍